docs: Add Client Agent commands to CLI docs (#5826)
hashicorp/boundary: website/content/docs/commands/client-agent/index.mdx

---
layout: docs
page_title: client-agent - Command
description: >-
  The "client-agent" command lets you interact with the Boundary Client Agent.
---

# client-agent

Command: `boundary client-agent`

The `client-agent` command lets you interact with the Boundary Client Agent.
You can check the Client Agent's status, pause it, resume it, or list the active transparent sessions the Client Agent is managing.

## Examples

The following command lets you check the status of the Client Agent to ensure it is running:

```shell-session
$ boundary client-agent status
```

The following command retrieves information about any sessions that the Client Agent is managing:

```shell-session
$ boundary client-agent sessions
```

You can temporarily disable the Client Agent by pausing it with the following command:

```shell-session
$ boundary client-agent pause
```

## Usage

<CodeBlockConfig hideClipboard>

```shell-session
Usage: boundary client-agent <subcommand> [options] [args]

  # ...

> Subcommands:
>     pause           Pauses the Client Agent when it is running.
>     resume          Resumes the Client Agent when it is paused.
>     sessions        Lists any active transparent sessions.
>     status          Gets status information for the Client Agent.

Reviewer: this is what I see when I run `boundary client-agent help` 

```suggestion
Subcommands:
    pause       Pauses the running boundary client agent
    resume      Resumes the paused boundary client agent
    sessions    List active transparent sessions managed by the client agent.
    status      Get the status information of the running boundary client agent
```

Author: Thanks! I applied this change.
